src/function/v2.ts

```diff
@@ -3,6 +3,17 @@ export type { Context } from '@netlify/serverless-functions-api'
 type Path = `/${string}`
 type HTTPMethod = 'GET' | 'POST' | 'PUT' | 'PATCH' | 'DELETE' | 'OPTIONS'
 type CronSchedule = string
+type RateLimitAlgorithm = 'sliding_window'
+type RateLimitAggregator = 'domain' | 'ip'
+type RateLimitAction = 'rate_limit' | 'rewrite'
+
+interface RateLimitConfig {
+  action?: RateLimitAction
+  aggregateBy?: RateLimitAggregator | RateLimitAggregator[]
+  algorithm?: RateLimitAlgorithm
```

Comment:
Why do we have this property if `sliding_window` is the only possible value?

Reply:
Great point, I've removed it!